site/zh/blog/new-in-devtools-107/index.md

```diff
@@ -27,72 +27,72 @@ draft: true
 
 <!-- Content starts here -->
 
-<!-- ## Customize keyboard shortcuts in DevTools {: #shortcuts } -->
+<!-- ## 客制化DevTools的键盘快捷键 {: #shortcuts } -->
```

Comment:
```suggestion
## 客制化 DevTools 的键盘快捷键 {: #shortcuts }
```
- Always add a space between English word and Chinese.
- Remove the `<!-- -->` because this is the syntax for making it a comment.

Please help to apply both the above for the whole article.